src/pages/docs/en/quickstart-guides/sveltekit.md

```diff
@@ -5,73 +5,40 @@ position: 3
 layout: "@docs"
 ---
 
-> If this is your first Space app take a look at the [getting started guide](/docs/en/introduction/first-app).
 
-If you don't have an existing SvelteKit app already you can create a new one by following the offical [SvelteKit instructions](https://kit.svelte.dev/docs/introduction#getting-started).
+> This guide assumes that you have a SvelteKit app that you want to run on Space. If you don't have a SvelteKit app, follow the instructions on the [SvelteKit docs](https://kit.svelte.dev/docs/creating-a-project) to create one.
 
-For Space to be able to run SvelteKit it needs to use the node adapter. This ensures that the app is built to be served by a NodeJS server, refer to the [SvelteKit Docs](https://kit.svelte.dev/docs/adapters#supported-environments-node-js) for more information.
 
-Install the `@sveltejs/adapter-node` dependency, and modify `svelte.config.js` to the following:
+## Configure your SvelteKit app
+To run a SvelteKit app on Space, you'll need to use the [`@sveltejs/adapter-node`](https://kit.svelte.dev/docs/adapter-node) adapter. If you already use the Node adapter, skip to the [next step](#run-on-space) section.
 
-```jsx
-import adapter from '@sveltejs/adapter-node';
-import preprocess from 'svelte-preprocess';
-
-/** @type {import('@sveltejs/kit').Config} */
-const config = {
-	// Consult https://github.com/sveltejs/svelte-preprocess
-	// for more information about preprocessors
-	preprocess: preprocess(),
 
-	kit: {
-		adapter: adapter()
-	}
-};
+First, install the `@sveltejs/adapter-node` dependency
 
-export default config;
+```sh
+npm install @sveltejs/adapter-node
 ```
 
-Sample `package.json`
+then, in your `svelte.config.js` file, replace the current adapter with the node adapter:
 
-```json
-{
-	"name": "my-app",
-	"version": "0.0.1",
-	"scripts": {
-		"dev": "vite dev",
-		"build": "vite build",
-		"preview": "vite preview",
-		"check": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json",
-		"check:watch": "svelte-kit sync && svelte-check --tsconfig ./tsconfig.json --watch"
-	},
-	"devDependencies": {
-		"@sveltejs/adapter-auto": "next",
-		"@sveltejs/kit": "next",
-		"@types/cookie": "^0.5.1",
-		"svelte": "^3.46.0",
-		"svelte-check": "^2.7.1",
-		"svelte-preprocess": "^4.10.6",
-		"tslib": "^2.3.1",
-		"typescript": "^4.7.4",
-		"vite": "^3.1.0"
-	},
-	"type": "module",
-	"dependencies": {
-		"@fontsource/fira-mono": "^4.5.0",
-		"@sveltejs/adapter-node": "^1.0.0-next.92",
-		"yarn": "^1.22.19"
-	}
-}
+```js
+import adapter from '@sveltejs/adapter-node';
+// rest of the file
```

Comment:
We could use a diff to make it more clear what you have to change:

```diff
- import adapter from '@sveltejs/adapter-auto';
+ import adapter from '@sveltejs/adapter-node';
import preprocess from 'svelte-preprocess';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	// Consult https://github.com/sveltejs/svelte-preprocess
	// for more information about preprocessors
	preprocess: preprocess(),

	kit: {
		adapter: adapter()
	}
};

export default config;
```

Reply:
We should use the default `vitePreprocess` rather than `preprocess from 'svelte-preprocess'` because it requires to be installed separately for the additional functionalities, so anyone just copying the config will get error on deployment.

```diff
- import adapter from '@sveltejs/adapter-auto';
+ import adapter from '@sveltejs/adapter-node';
- import preprocess from 'svelte-preprocess';
+ import { vitePreprocess } from '@sveltejs/kit/vite';

/** @type {import('@sveltejs/kit').Config} */
const config = {
	// Consult https://github.com/sveltejs/svelte-preprocess
	// for more information about preprocessors
-	preprocess: preprocess(),
+	preprocess: vitePreprocess(),

	kit: {
		adapter: adapter()
	}
};

export default config;
```

Reply:
Yeah, I would include the diff.

While brevity is good, helpful to see what changed and have a copy-pastable.

Reply:
later, @xeust